test/reporter/test-reporter-util.js

```diff
@@ -79,6 +79,7 @@ test('util.sanitizeOutput', function (t) {
   var raw = fs.readFileSync(carriageReturnPath, 'utf-8');
   var expected = fs.readFileSync(carriageReturnExpectedPath, 'utf-8');
   var result = util.sanitizeOutput(raw, '#');
-  t.equals(result, expected, 'there should be a # on every line');
+  result += '\n';
```

Comment:
I'm not exactly following. Why would your text editor require us to add a newline to the test that didn't exist in the past?